### PR TITLE
docs: improve Restoring a Bareos Server section

### PR DESCRIPTION
**Pull request by UraniumDonut**
- Added a point about importing the Catalog Backup.
- Changed the next point to be syntactially correct.

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Required backport PRs have been created
- [x] Correct milestone is set

##### Source code quality
- [x] Source code changes are understandable
- [X] Required documentation changes are present and part of the PR

### CODE REVIEW

File: docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst [hunk 1]
```diff
@@ -167,7 +167,9 @@ Consider the following steps:
 
 -  If you have a valid saved Bootstrap file as created for your damaged machine with WriteBootstrap, use it to restore the files to the damaged machine, where you have loaded a static Bareos File daemon using the Rescue disk). This is done by using the restore command and at the yes/mod/no prompt, selecting mod then specifying the path to the bootstrap file.
 
--  If you have the Bootstrap file, you should now be back up and running, if you do not have a Bootstrap file, continue with the suggestions below.
+-  After the Catalog is restored, it should be located in (:file:`./var/lib/bareos/bareos.sql`) relative to the restore location. After stopping the bareos-dir service, his file then needs to be loaded into the PostgreSQL database. (For example by running ``sudo -u bareos psql bareos bareos -f /var/lib/bareos/bareos.sql``)
```
Comment (bruno-at-bareos) [inline docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst:170]: I would slightly fix the comment by this formulation.

```suggestion
-  After the Catalog is restored, it should be located in (:file:`./var/lib/bareos/bareos.sql`) relative to the restore location. After stopping the bareos-dir service, this file then needs to be loaded into the PostgreSQL database. (For example by running ``sudo -u bareos psql bareos bareos -f ./var/lib/bareos/bareos.sql`` or ```su postgres -c "psql bareos -f ./var/lib/bareos/bareos.sql``)
```

**Review by bruno-at-bareos — CHANGES_REQUESTED** [review]
Thanks for your suggestion, I've a slightly different fix  you may want to apply.

**Comment by bruno-at-bareos** [comment]
Also now the build doc has tried there's some failures you want to fix

```

**Comment by UraniumDonut** [comment]
Thanks for the hints!
As I have no idea how to edit existing commit messages, I will re-open a new PR with the new changes and proper line endings.

**Comment by bruno-at-bareos** [comment]
Reopened, fixes will goes there as all the other parameters are already in place. (tagging project labels etc)

**Comment by bruno-at-bareos** [comment]
We are working on a way to allow external contributor to have access to build result, I will take care of the message after your submission.

**Comment by bruno-at-bareos** [comment]
We have this section into the documentation that may help 
https://docs.bareos.org/master/DeveloperGuide/gitworkflow.html#commit-message-guideline

For example your commit to the rst can have as subject
```
doc: improve section recovering Bareos server section
```
With a body like 
```
Add examples how to load the restored database ascii dump as catalog.
``` 
So the subject is short enough (around 50 char) but explicit for the changelog.
(you may want to consult the existing to find similar commit)

Once you have it done, you should also modify the title pr here so it can be also used during the merge.
Don't hesitate to ask help if needed.

**Comment by UraniumDonut** [comment]
I think I managed to change it properly. Thanks for all the tips, I already learned a lot more about git!

**Comment by UraniumDonut** [comment]
Still failing :/

**Comment by bruno-at-bareos** [comment]
> Build still failing :/

No in fact, the final review is now needed (which I will take care of today).

**Comment by bruno-at-bareos** [comment]
> > Build still failing :/
> 
> No in fact, the final review is now needed (which I will take care of today).

And I will fix the warning of our pr-tool
Commit checks failed:
 	b2401123f docs: improve section recovering Bareos server section: body contains line longer 72 chars

If you want to contribute more, it will be interesting to setup a dev environment (see documentation it should be complete), you will find our tools in the devtools subdir.

**Review by bruno-at-bareos — APPROVED** [review]
Thanks for the work, Approved.

**Comment by UraniumDonut** [comment]
Thanks for the help!

**Pull request merged**